docs/spec.md

```diff
@@ -8,13 +8,16 @@ A CLI (Command Line Interface) tool that exports issues in a GitHub repository t
 
 ### GitHub CLI
 
-The official [GitHub CLI](todo-url) allows:
-- one
-- two
+The official [GitHub CLI](https://cli.github.com) allows:
+- Clone Repo
+- Create Repo
+- Create Issues
+- Checkout PR
 
-### Other tool
+### Other tools
 
-...
+### [OAuth app](https://github-issues-to-excel.netlify.app) 
+- [Github link](https://github.com/Joshua-rose/github-issues-to-excel/tree/master/src)
 
 ## Features
 
```

Comment:
The only real feature here is the third one (Build/export xls files) - but that is the scope of the whole project anyway. What we need to do is expand on this - e.g.

## Features

### Basic project report

> Generate a report of all issues in the repository.

Command
```shell
gh-xlsx report basic
```

Options:
- `--all` Shows both open and close issues
- `--json` Saves the report as JSON file (instead of XLSX file)

Fields (columns)
- Number
- Title
- Creation On
- Created By
- Assignee
- Last Updated On
- Status (closed, open)

### Tag wise project report

### ...